container.go

```diff
@@ -50,7 +50,7 @@ type Container interface {
 	Stop(context.Context, *time.Duration) error                                    // stop the container
 
 	// Terminate stops and removes the container and its image if it was built and not flagged as kept.
-	Terminate(ctx context.Context) error
+	Terminate(ctx context.Context, opts ...TerminateOption) error
```

Comment:
As this is breaking change, the PR title must be updated to `feat(termination)!: ...`

See https://golang.testcontainers.org/contributing/#submitting-your-changes

Reply:
thank you sure, gonna update it. i didn't think of it as breaking change, given the variadic function will make `opts ` optional

Reply:
Indeed, it could slip into a breaking change in two cases: somebody implementing their own container, i.e. modules, as you already spotted in the etcd module, and if somebody has the Terminate function assigned to a variable (see https://go.dev/play/p/5LsWTXgv_iU). Very rare thought

Reply:
gotcha, good point 👍🏼